javascript/react_js/react_testing_part_two.md

```diff
@@ -157,4 +157,5 @@ This section contains questions for you to check your understanding of this less
 
 This section contains helpful links to other content. It isn't required, so consider it supplemental.
 
-* It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum.
+* Check out this [React testing library tutorial](https://www.youtube.com/playlist?list=PL4cUxeGkcC9gm4_-5UsNmLqMosM-dzuvQ) for a comprehensive overview the last two lessons.
```

Comment:
I think this is a fine addition. There's a video on Integration Tests in the playlist which is out of scope in the curriculum, we should mention it here.

Reply:
@justEstif were there any particular videos out of this playlist that you were wanting to include? I'm going to give this PR another week.

Reply:
I don't think this is necessary anymore.

Reply:
@justEstif do you want us to close the ticket?

Reply:
Yes.